package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "notifications-node-client",
-  "version": "4.7.1",
+  "version": "4.7.2",
+  "homepage": "https://docs.notifications.service.gov.uk/node.html#node-js-client-documentation",
```

Comment:
Optional, remove the unneeded anchor in the URL as it's still just going to the top of the page